app/v1/services/ListForeignPropertiesService.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.services
+
+import cats.data.EitherT
+import cats.implicits._
+import javax.inject.{Inject, Singleton}
+import uk.gov.hmrc.http.HeaderCarrier
+import utils.Logging
+import v1.connectors.ListForeignPropertiesConnector
+import v1.controllers.EndpointLogContext
+import v1.models.errors._
+import v1.models.request.listForeignProperties.ListForeignPropertiesRequest
+import v1.support.DesResponseMappingSupport
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class ListForeignPropertiesService @Inject()(connector: ListForeignPropertiesConnector) extends DesResponseMappingSupport with Logging {
+
+  def listForeignProperties(request: ListForeignPropertiesRequest)(
+    implicit hc: HeaderCarrier,
+    ec: ExecutionContext,
+    logContext: EndpointLogContext): Future[ListForeignPropertiesServiceOutcome] = {
+
+    val result = for {
+      desResponseWrapper <- EitherT(connector.listForeignProperties(request)).leftMap(mapDesErrors(desErrorMap))
+    } yield desResponseWrapper
+
+    result.value
+  }
+
+  private def desErrorMap =
+    Map(
+      "INVALID_TAXABLE_ENTITY_ID" -> NinoFormatError,
+      "FORMAT_BUSINESS_ID" -> BusinessIdFormatError,
+      "FORMAT_SUBMISSION_ID" -> SubmissionIdFormatError,
+      "NOT_FOUND" -> NotFoundError,
+      "SUBMISSION_ID_NOT_FOUND" -> SubmissionIdNotFoundError,
+      "SERVER_ERROR" -> DownstreamError,
+      "SERVICE_UNAVAILABLE" -> DownstreamError
```

Comment:
`FORMAT_SUBMISSION_ID` and `SUBMISSION_ID_NOT_FOUND` shouldn't be here as we don't use a submissionId